lib/stdlib/doc/src/peer.xml

```diff
@@ -0,0 +1,319 @@
+          </item>
+          <tag><c>report_reason</c></tag>
+          <item>
+            <p>When set to false (default), peer node termination changes the
+              status reported by controlling process. When set to true,
+              peer shutdown causes controlling process to exit with the
+              <c>Reason</c> of the peer node termination.</p>
+          </item>
+          <tag><c>progname</c></tag>
+          <item>
+            <p>Path to "erl", by default taken from <c>init:get_argument(progname)</c>.
+              If progname is not known, <c>peer</c> makes best guess given current
+              ERTS version.
+            </p>
+            <p>
+              Useful for testing backwards compatibility with previous releases,
+              or when Erlang installation location is missing from the <c>PATH</c>,
+              or when starting peers on a host with different operating system.
+            </p>
+          </item>
+          <tag><c>connection</c></tag>
+          <item>
+            <p>Alternative connection specification.</p>
+          </item>
+          <tag><c>args</c></tag>
+          <item>
+            <p>Extra command line arguments to pass to "erl". Arguments are passed as is,
+            no escaping or quiting is needed or accepted.</p>
+          </item>
+          <tag><c>env</c></tag>
+          <item>
+            <p>List of environment variables with their values. This list is applied
+            to a locally started executable. If <c>remote</c> is used or implied, these
+            variables are applied to <c>rsh</c> itself, but not the remote peer. If you
+            need to change environment of the remore peer, use adjust <c>args</c> to
+            contain <c>-env ENV_KEY ENV_VALUE</c>.</p>
+          </item>
+          <tag><c>wait_boot</c></tag>
+          <item>
+            <p>Specifies how long start/start_link timeout.</p>
+          </item>
+          <tag><c>shutdown</c></tag>
+          <item>
+            <p>Specifies peer node stopping behaviour, see
+              <seemfa marker="#stop/1"><c>stop()</c></seemfa>.</p>
+          </item>
+        </taglist>
+      </desc>
+    </datatype>
+    <datatype>
+      <name name="peer_state"/>
+      <desc><p>Peer node state. </p></desc>
+    </datatype>
+    <datatype>
+      <name name="connection"/>
+      <desc><p>Alternative connection between the origin and the peer. When the connection
+        closes, peer node is terminated. If <c>report_reason</c> startup flag is set to true,
+        controlling process on the origin node exits with corresponding reason, effectively
+        providing two-way link.</p>
+        <p>When <c>connection</c> is set to a port number, origin starts listening on
+          the requested TCP port, and peer node connects to the port. When it is set to
+          an <c>{IP, Port}</c> tuple, origin listens only on the specified IP. Port can
+          be set to 0 for automatic selection.
+        </p>
```

Comment:
Would this be similar to using a peer node started by `test_server:start_node()` with the `peer` option?